geode-core/src/main/java/org/apache/geode/internal/monitoring/executor/FunctionExecutionPooledExecutorGroup.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.monitoring.executor;
+
+import org.apache.geode.internal.monitoring.ThreadsMonitoring;
+
+public class FunctionExecutionPooledExecutorGroup extends AbstractExecutor {
+
+  public static final String GRPNAME = "FunctionExecutionPooledExecutor";
+
+
+  public FunctionExecutionPooledExecutorGroup(ThreadsMonitoring tMonitoring) {
+    super(tMonitoring);
+    super.setGrpName(GRPNAME);
```

Comment:
why are you calling super.setGrpName()?  Just call setGrpName()?  I wish you'd change all of the "grp" stuff to just spell out the word "group".